lib/src/dartdoc_options.dart

```diff
@@ -168,7 +168,7 @@ class ToolDefinition {
   String toString() {
     final commandString =
         '${this is DartToolDefinition ? '(Dart) ' : ''}"${command.join(' ')}"';
-    if (setupCommand == null) {
+    if (setupCommand == null || setupCommand.isEmpty) {
```

Comment:
As this is a debug string, I think it's functional that we have different output for an empty string vs a null.

Reply:
Makes sense!